docs/dapp-staking/for-devs/requirements.md

```diff
@@ -8,7 +8,9 @@ sidebar_position: 1
 
 Are you interested to join dApp staking on Shiden or Astar? Read carefully to see if you meet the requirements:
 
-> Astar is aiming to become a DAO in a couple of years. While moving towards becoming a DAO our dApp staking onboarding also includes passing our council. The process is to create your proposal to join dApp staking also on our [forum](https://forum.astar.network/c/builders-program/proposals/16) after meeting the requirements below.
+:::info
+Astar is aiming to become a DAO in a couple of years. While moving towards becoming a DAO our dApp staking onboarding also includes passing our council. The process is to create your proposal to join dApp staking also on our [forum](https://forum.astar.network/c/builders-program/proposals/16) after meeting the requirements below.
```

Comment:
Correct link to the forum should be:
https://forum.astar.network/c/initiatives/dapp-staking-applications/21

Reply:
This change was already made in the main branch so merging should fix the issue.